packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -108,11 +112,50 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   /* @conditional-compile-remove(rooms) */
   mobileWithPreview = mobileWithPreview && rolePermissions.cameraButton;
 
+  /* @conditional-compile-remove(call-readiness) */
+  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
+  /* @conditional-compile-remove(call-readiness) */
+  const onLightDismissTriggered = (): void => {
+    // do nothing here
+    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
+  };
+  /* @conditional-compile-remove(call-readiness) */
+  const drawerStyle: _DrawerSurfaceStyles = {
+    root: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      // apply zindex = 99 so drawer appear over device buttons and other components in the config page
+      zIndex: 99
+    }
+  };
+
   return (
     <Stack className={mobileView ? configurationContainerStyleMobile : configurationContainerStyleDesktop}>
       <Stack styles={bannerNotificationStyles}>
         <ErrorBar {...errorBarProps} />
       </Stack>
+
+      {
+        /* @conditional-compile-remove(call-readiness) */
+        mobileView && isDrawerShowing && (
+          <_DrawerSurface onLightDismiss={onLightDismissTriggered} styles={drawerStyle}>
+            <DomainPermissions
+              appName={'app'}
+              onTroubleshootingClick={() => console.log('clicked trouble shooting link')}
+              onAllowAccessClick={async () => {
+                await adapter.askDevicePermission({ video: true, audio: true });
+                adapter.queryCameras();
+                adapter.queryMicrophones();
+                adapter.querySpeakers();
+                setIsDrawerShowing(false);
```

Comment:
for a future PR but flagging here for posterity; if they have denied access to the permissions we want to show the appropriate message here